predictoor/pdr-faq.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Predictoor FAQ
+description: Frequently Asked Questions about Predictoor
+---
+
+## Predictoor FAQ
+
+### Getting Started
+
+<details>
+
+<summary>How can users begin experimenting with Predictoor?</summary>
+
+You can start by playing with [predictoor.ai](https://, then click "run bots" and go through [Predictoor](https://github.com/oceanprotocol/pdr-backend/blob/main/READMEs/predictoor.md) or [Trader](https://github.com/oceanprotocol/pdr-backend/blob/main/READMEs/trader.md) README. This [blog](https://blog.oceanprotocol.com/meet-predictoor-accountable-accurate-prediction-feeds-8b104d26a5d9#048b) "How to Earn" section has details.  
```

Comment:
Missing rest of url. `[predictoor.ai](https://` -->  `[predictoor.ai](https://predictoor.ai)`